frontend/src/components/app/App.tsx

```diff
@@ -41,7 +100,7 @@ const App: React.FC = () => {
               <AppBar position="fixed">
                 <Toolbar>
                   <div className="brand">Powered By
-                    <a href="https://www.headstorm.com">HEADSTORM, LLC.</a>
+                      <a href="https://www.headstorm.com">HEADSTORM, LLC.</a>
```

Comment:
This seems like an unnecessary change (and is incorrect indention by the looks of it)